### ISSUE TITLE
How to install (ELI5) ?

### ISSUE THREAD

**Issue by metalshanked** [CybercentreCanada/jupyterlab-sql-editor#76]
Hi, 

Thanks for this promising extension. However, i am a newbie and not able to get this installed.
I did a `pip install jupyterlab-lsp jupyterlab-sql-editor` and `npm install -g sql-language-server` 
and I do see these installed doing a `pip list`
![image](https://user-images.githubusercontent.com/57646596/216750010-b8f50aa9-a0e1-4a83-a0ab-0a1f6670ccff.png)

However, when i do `%%sparksql` , it cant find it

![image](https://user-images.githubusercontent.com/57646596/216750041-4662199a-cbef-49f5-af65-01abc942a8e3.png)

![image](https://user-images.githubusercontent.com/57646596/216750642-ce1c97ff-32fe-4189-b6f4-eb550c6862ae.png)

Can anyone please advice?

Thanks in advance!

**Comment by metalshanked** [comment]
Update: After some trial and error, figured out that the fully qualified module path is required like so
![image](https://user-images.githubusercontent.com/57646596/216782651-614e4a90-e6c0-4047-a8dc-9003b3aae916.png)

Also,
if you want to start it automatically, then below can be added to your IPython Config Profile
`~/.ipython/profile_default/ipython_config.py `

```
# This is the ipython_config.py file
# get the config
c = get_config()

# pre-load the sparksql magic
c.InteractiveShellApp.extensions = [
    'jupyterlab_sql_editor.ipython_magic.trino', 'jupyterlab_sql_editor.ipython_magic.sparksql'
]

# pre-configure the SparkSql magic.
c.SparkSql.limit=20
c.SparkSql.cacheTTL=3600
c.SparkSql.outputFile='/tmp/sparkdb.schema.json'
c.SparkSql.catalogs='default'

# pre-configure the Trino magic.
import trino
c.Trino.auth=trino.auth.BasicAuthentication(TRINO_USER, TRINO_PASSWORD)
c.Trino.user=TRINO_USER
c.Trino.host=TRINO_HOST
c.Trino.port=443
c.Trino.httpScheme='https'
c.Trino.verify=False
c.Trino.cacheTTL=3600
c.Trino.outputFile='/tmp/trinodb.schema.json'
c.Trino.catalogs="system,tpch"

# pre-configure to display all cell outputs in notebook
from IPython.core.interactiveshell import InteractiveShell
InteractiveShell.ast_node_interactivity = 'all'
```

**Comment by cccs-jc** [comment]
Sorry about that. I see that our documentation is inaccurate. 

https://github.com/CybercentreCanada/jupyterlab-sql-editor/blob/5feb9064627570b6ca21c05680d2cdd704a7402f/CONTRIBUTING.md#pre-configure-magics-optional

We will update it.

**Comment by cccs-nik** [comment]
Updated, sorry for the mistake.
